package.json

```diff
@@ -85,8 +85,8 @@
     "html-webpack-plugin": "2.16.0",
     "image-webpack-loader": "1.7.0",
     "jasmine-reporters": "2.1.1",
-    "jest-cli": "12.1.1",
-    "jest-webpack-alias": "2.3.0",
+    "jest-cli": "^16.0.2",
+    "jest-webpack-alias": "^3.3.0",
```

Comment:
@philipnrmn can you remove the `^`.
This way we avoid the "It works on my machine" thing.